data/sql/updates/pending_db_world/add_scriptnames_to_bosses.sql

```diff
@@ -0,0 +1,12 @@
+--
+UPDATE `creature_template`
+SET `Scriptname` = 'boss_mennu_the_betrayer'
+WHERE `entry` = 17941 AND `difficulty_entry_1` = 19893;
+
+UPDATE `creature_template`
+SET `Scriptname` = 'boss_quagmirran'
+WHERE `entry` = 17942 AND `difficulty_entry_1` = 19894;
+
+UPDATE `creature_template`
+SET `Scriptname` = 'boss_rokmar_the_crackler'
+WHERE `entry` = 17991 AND `difficulty_entry_1` = 19895;
```

Comment:
No need to use multiple lines here or the difficulty_entry_1 because the entries are immutable in creature_template, but it's no big deal as well.

3 things though:

1. Remember to also set the AIName to '' to remove SmartAI
2. Use only 1 .sql file
3. When deleting from smart_scripts, remember to specify the source_type (0 for creatures)

Reply:
should be fixed now!